tests/unary_expression/urnary_expression.js

```diff
@@ -4,3 +4,5 @@ x--;
 -+1;
 x + + + + 1;
 x + (+ (+ (+ 1)))
+x * +y;
++x * y;
```

Comment:
@bakkot in the end of `tests/unary_expression/urnary_expression.js` (by the way, there's a typo).